package.json

```diff
@@ -98,7 +98,7 @@
     "build:min": "cross-env FILE=min rollup -c",
     "clean": "rimraf build/*",
     "copyright": "node -r esm license-update",
-    "dist:copy": "cpy build/cytoscape.umd.js build/cytoscape.min.js build/cytoscape.cjs.js build/cytoscape.esm.js dist",
+    "dist:copy": "cpy build/cytoscape.umd.js build/cytoscape.min.js build/cytoscape.cjs.js build/cytoscape.esm.js build/cytoscape.esm.min.js build/cytoscape.esm.min.js.map dist",
```

Comment:
Let's not include the map file.  It's more useful for debugging than production.  Many packages don't include sourcemaps for prod, e.g. https://unpkg.com/browse/lodash@4.17.15/

Reply:
There should also be something like a `build:esm.min` script